src/components/shared/Validations.tsx

```diff
@@ -25,7 +25,7 @@ export const numberValidationNames = [
   'exclusiveMaximum',
 ];
 
-const exampleValidationNames = ['examples', 'example', 'x-example'];
+const exampleValidationNames = ['examples'];
```

Comment:
`http-spec` will try to produce `examples` at all times now, thus there's no need to handle `example` and `x-example` separately.